src/words/job.rs

```diff
@@ -1,5 +1,4 @@
-
-pub(crate) const JOBS: [&'static str;69] = [
+pub const JOBS: [&'static str; 69] = [
```

Comment:
This makes it public to everyone, this alright?

Reply:
sure that's fine